File: app/src/main/java/com/example/myanimedata/api/MangaResult.java
package com.example.myanimedata.api;

import com.google.gson.annotations.SerializedName;

public class MangaResult {
    @SerializedName("mal_id")
    private final int id;

    @SerializedName("title")
    private final String title;

    @SerializedName("images")
    private final ImageResult imageResults;

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public ImageResult getImageResults() {
        return imageResults;
    }

    public MangaResult(int id, String title, ImageResult imageResults) {
        this.id = id;
        this.title = title;
        this.imageResults = imageResults;
    }
}

File: app/src/main/java/com/example/myanimedata/api/MangaResult.kt
package com.example.myanimedata.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class MangaResult(
    @field:SerializedName("mal_id") val id: Int? = 0,
    @field:SerializedName("title") val title: String? = null,
    @field:SerializedName("images") val imageResults: ImageResult? = null
) : Parcelable